src/timeline.js

```diff
@@ -730,8 +730,10 @@ export default class Timeline extends React.Component {
    * Only calls back if a new snap time is reached
    */
   mouseMoveFunc(e) {
+    const {componentId} = this.props;
+    const leftOffset = document.querySelector(`.rct9k-id-${componentId} .parent-div`).getBoundingClientRect().left;
```

Comment:
@scoob do you know if having a DOM selector in a move mouse function will be super bad for performance?